Sources/Sentry/SentrySerialization.m

```diff
@@ -107,6 +107,27 @@ + (NSData *_Nullable)dataWithEnvelope:(SentryEnvelope *)envelope
     return envelopeData;
 }
 
++ (NSString *)urlEncodedDictionary:(NSDictionary *)dictionary
+{
+    NSMutableArray *itens = [[NSMutableArray alloc] initWithCapacity:dictionary.count];
+
+    NSMutableCharacterSet *allowedSet = [NSCharacterSet.alphanumericCharacterSet mutableCopy];
+    [allowedSet addCharactersInString:@"-_"];
+
+    for (id key in dictionary.allKeys) {
+        id value = dictionary[key];
+        NSString *keyDescription =
+            [[key description] stringByAddingPercentEncodingWithAllowedCharacters:allowedSet];
+        NSString *valueDescription =
+            [[value description] stringByAddingPercentEncodingWithAllowedCharacters:allowedSet];
+        [itens addObject:[NSString stringWithFormat:@"%@=%@", keyDescription, valueDescription]];
+    }
+
+    return [[itens sortedArrayUsingComparator:^NSComparisonResult(NSString *obj1, NSString *obj2) {
+        return [obj1 compare:obj2];
+    }] componentsJoinedByString:@"&"];
```

Comment:
should this be `,` instead of `&`?

See https://www.w3.org/TR/baggage/#definition and https://www.w3.org/TR/baggage/#examples-of-http-headers

> userId=alice,serverNode=DF%2028,isProduction=false

Reply:
Ow, I see. So is not like a http form or query encode.